circle.yml

```diff
@@ -0,0 +1,106 @@
+machine:
+  timezone:
+    Europe/Berlin
+  services:
+    - docker
+  environment:
+    DOCKERFLAGS: --rm=false
+dependencies:
+  pre:
+  - sudo apt-get update
+  - sudo apt-get install clang
+  - sudo apt-get install build-essential libtool autotools-dev automake checkinstall check yasm
+  - sudo apt-get install pkg-config
+  - java -version ; exit 0
+  - gcc --version ; exit 0
+  - clang --version ; exit 0
+
+compile:
+  pre:
+    - echo "dummy"
+  override:
+    - cat /etc/apt/sources.list
+    - ls -al /etc/apt/sources.list.d
+    - docker info
+    - make
+    - make -C ghc-android build-arm > $CIRCLE_ARTIFACTS/make_out.txt 2>&1 ; if [ $? -eq 0 ]; then touch ~/ok.txt ; fi ; touch ~/ready.txt :
+        background: true
+
+    - ready_="0";
```

Comment:
Lots of repetition. Perhaps put this code in a separate script and call it from here?